spec/features/users/user_can_search_rdv_on_rdv_mairie_spec.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+describe "User can search rdv on rdv mairie" do
+  let(:now) { Time.zone.parse("2021-12-13 8:00") }
+
+  def json_response
+    JSON.parse(page.html)
+  end
+
+  before do
+    default_url_options[:host] = "http://www.rdv-mairie-test.localhost"
+  end
+
+  around do |example|
+    previous_auth_token = ENV["ANTS_API_AUTH_TOKEN"]
+    ENV["ANTS_API_AUTH_TOKEN"] = ""
+
+    example.run
+
+    ENV["ANTS_API_AUTH_TOKEN"] = previous_auth_token
+  end
+
+  before do
+    travel_to(now)
```

Comment:
vu que `now` est juste utilisé ici, on peut limiter les indirections et inliner 
```suggestion
    travel_to(Time.zone.parse("2021-12-13 8:00"))
```

Reply:
`now` est utilisé également au niveau de la ligne juste en dessous (25)

Reply:
ah oui, autant pour moi !